keycloak_backend/api/auth.py

```diff
@@ -21,6 +22,11 @@ def get_domain():
   return str(os.getenv('BACKEND_DOMAIN'))
```

Comment:
This returns `"None"` when the env variable is not defined.

```suggestion
  return os.getenv('BACKEND_DOMAIN', '')
```